submissions/andrewklmn/js-movie-seat-booking/modules/ticket-updater.js

```diff
@@ -11,6 +11,78 @@ import {
 
 import getSelectedSeats from './seat-selector';
 
+const MAX_DETAILED_SEATS_IN_TICKET = 6;
+const MAX_LENGTH_OF_SIMPLE_SEAT_LIST = 4;
+
+function getShortedSeatsInRowRecord(seats) {
+  if (seats.length < MAX_LENGTH_OF_SIMPLE_SEAT_LIST) {
+    return seats.join(',');
+  }
+
+  const sequences = [[seats[0]]];
+  let currentSequenceIndex = 0;
+
+  let i = 1;
+  while (i < seats.length) {
+    if (seats[i] - seats[i - 1] === 1) {
+      sequences[currentSequenceIndex][1] = seats[i];
+    } else {
+      currentSequenceIndex += 1;
+      sequences.push([seats[i]]);
+    }
+    i += 1;
+  }
+
+  return sequences.map((sequence) => {
+    if (sequence.length === 1) {
+      return sequence[0];
+    }
+    if (sequence[1] - sequence[0] === 1) {
+      return sequence.join(',');
+    }
+    return sequence.join('-');
+  }).join(',');
+}
+
+function getSeatPosition(seatElement) {
+  const seatPosition = seatElement.id.split('_');
+  return {
+    row: seatPosition[1],
+    seat: seatPosition[2],
+  };
```

Comment:
```suggestion
  const [row, seat] = seatElement.id.split('_');
  return {
    row,
    seat
  }
```
Don't forget about [Destructuring assignment](https://javascript.info/destructuring-assignment) )